src/Bootstrap/LoadConfiguration.php

```diff
@@ -32,7 +32,9 @@ public function bootstrap(Application $app): void
 
         $app->detectEnvironment(
             function () use ($app) {
-                return $app['config']->get('app.production', true) ? 'production' : 'development';
+                $isProduction = $app['config']->get('app.production', true);
```

Comment:
Can we flip the logic so if you define `app.env` you don't have to define `app.production`?

For example:

```php
return $app['config']->get('app.env', function () use ($app) {
    return $app['config']->get('app.production', true) ? 'production' : 'development;
});
```

This would make it easier to phase out `app.production` in the future.  With the current code I would need to set both `app.production` and `app.env` to use a `testing` environment.

Reply:
For me this code go to the next major version. IMO we could remove the `app.production`, and read it from the `.env`. just like laravel.